### PR TITLE
[MQTT] clearly set null when closing connection

### PR DESCRIPTION
**Pull request by jaeyun-jung**
Clearly set connection state and handle when closing MQTT connection.

### CODE REVIEW

**Review by songgot — APPROVED** [review]
LGTM

**Review by gichan-jang — APPROVED** [review]
LGTM

File: src/libnnstreamer-edge/nnstreamer-edge-mqtt-paho.c [hunk 1]
```diff
@@ -258,6 +258,8 @@ nns_edge_mqtt_close (nns_edge_broker_h broker_h)
     MQTTAsync_destroy (&handle);
   }
 
+  bh->mqtt_h = NULL;
```
Comment (myungjoo) [inline src/libnnstreamer-edge/nnstreamer-edge-mqtt-paho.c:261]: What about ```bh->connected``` ?

Comment (jaeyun-jung) [reply src/libnnstreamer-edge/nnstreamer-edge-mqtt-paho.c:261]: The struct for paho-mqtt does not contain connected flag.

**Review by myungjoo — APPROVED** [review]

**Pull request merged**